### ISSUE TITLE
slow amuleweb webgui connection

### ISSUE THREAD

**Issue by gattino973** [tchabaud/dockerfiles#6]
Hi, now I had another problem. Once connected the connection immediately becomes slow and I cannot navigate both with ameleweb and amulegui. Has anyone else had this problem?

**Comment by t-chab** [comment]
Hi,

What do you mean by "slow" ? The web interface is slow to respond ?

**Comment by gattino973** [comment]
yes, as I connect, immediately after, both amuleweb and amulegui become irresponsive

**Comment by t-chab** [comment]
Could you try with the latest image available ?

Can you post the output of the following command ?

```bash
docker logs [your amule container name]
```

**Comment by gattino973** [comment]
ok, I downloaded the last image, I deleted the config folder and launched it as first time. I connected with the web interface and was responsive. After I started downloading a file, the interface became slower until it became unusable

The output of the command you have requested:

root@NSA55L:~# docker logs aMule
Group 100 already exists. Won't be created.
Group users with GID 100 will be used as amule group.
No GUI password specified, using generated one: QDizpwoVe2Le7tgsDdDq3tHczNOGSTIGSfrRupoJez6hjuRgtp3l04x9fyNmcMZp
/home/amule/.aMule/amule.conf file NOT found. Generating new default configuration ...
/home/amule/.aMule/amule.conf successfullly generated.
/home/amule/.aMule/remote.conf file NOT found. Generating new default configuration ...
/home/amule/.aMule/remote.conf successfullly generated.
Using default theme for web UI.
 2019-04-16 17:11:29: Logging to stdout enabled
 2019-04-16 17:11:29: Initialising aMuleD 2.3.2 compiled with wxBase(GTK2) v2.8.12 (Snapshot: rev. g5d9c1f458)
 2019-04-16 17:11:29: Checking if there is an instance already running...
 2019-04-16 17:11:29: No other instances are running.
!2019-04-16 17:11:29: ERROR: Info  --- This is the first time you run aMule 2.3.2 ---
!2019-04-16 17:11:29: More information, support and new releases can found at our homepage,
!2019-04-16 17:11:29: at www.aMule.org, or in our IRC channel #aMule at irc.freenode.net.
!2019-04-16 17:11:29: Feel free to report any bugs to http://forum.amule.org
 2019-04-16 17:11:29: No 'cryptkey.dat' file found, creating.
 2019-04-16 17:11:29: *** TCP socket (ECServer) listening on 0.0.0.0:4712
 2019-04-16 17:11:29: Created Server UDP-Socket at port 4665
 2019-04-16 17:11:29: ListenSocket: Ok.
 2019-04-16 17:11:29: Created Client UDP-Socket at port 4672
 2019-04-16 17:11:29: Loading server.met file: /home/amule/.aMule/server.met
 2019-04-16 17:11:29: Server.met file not found!
 2019-04-16 17:11:29: Loading temp files from /temp.
 2019-04-16 17:11:29: All PartFiles Loaded.
 2019-04-16 17:11:29: No part files found
 2019-04-16 17:11:29: Found 0 known shared files, 7 unknown
!2019-04-16 17:11:30: web server running on pid 34
 2019-04-16 17:11:30: amuled: OnInit - starting timer
 2019-04-16 17:11:30: Loading IP filters 'ipfilter.dat' and 'ipfilter_static.dat'.
 2019-04-16 17:11:30: Loaded 0 IP-ranges from '/home/amule/.aMule/ipfilter.dat'.
 2019-04-16 17:11:30: Loaded 0 IP-ranges from '/home/amule/.aMule/ipfilter_static.dat'.
 2019-04-16 17:11:30: Downloaded 621 bytes
 2019-04-16 17:11:30: IP filter is ready
!2019-04-16 17:11:30: Connecting
!2019-04-16 17:11:30: No valid servers to which to connect found in server list
!2019-04-16 17:11:30: No contacts found, please bootstrap, or download a nodes.dat file.
!2019-04-16 17:11:30: Kad started.
 2019-04-16 17:11:30: Loading server.met file: /home/amule/.aMule/server.met.download
!2019-04-16 17:11:30: 11 servers in server.met found
 2019-04-16 17:11:30: Finished downloading the server list from http://gruk.org/server.met.gz
!2019-04-16 17:11:30: Failed to connect to all obfuscated servers listed. Making another pass without obfuscation.
 2019-04-16 17:11:30: New external connection accepted
 2019-04-16 17:11:30: Connecting client: aMuleweb 2.3.2
 2019-04-16 17:11:30: Access granted.
 2019-04-16 17:11:30: Connecting to eDonkey Server No3 (91.226.212.11 - 91.226.212.11:2442)
 2019-04-16 17:11:30: Connected to eDonkey Server No3 (91.226.212.11:2442)
 2019-04-16 17:11:32: Connecting to ProphetCraft (114.55.28.216 - 114.55.28.216:10032)
!2019-04-16 17:11:32: Connection established on: eDonkey Server No3
!2019-04-16 17:11:32: Connected to eDonkey Server No3 with HighID
 2019-04-16 17:11:32: New clientid is 1595075586
 2019-04-16 17:11:32: ServerMessage: server version 17.15 (lugdunum)
 2019-04-16 17:11:32: ServerMessage: eDonkey server do not host any file
!2019-04-16 17:13:27: ExternalConn: adding link 'ed2k://'.
!2019-04-16 17:13:27: Invalid eD2k link! ERROR: Not a valid ed2k-URI
!2019-04-16 17:13:27: ExternalConn: adding link 'ed2k://|file|The.P****ge.1x09.I.D****i.ITA.DLMux.x264-UBi.mkv|390976900|0F1****BA9250EACFED****E0C2181AC|h=AAMGCVJZXXE6C****YZRS5Q446ESLN47|/'.

**Comment by t-chab** [comment]
On which hardware do you run this container ?

**Comment by gattino973** [comment]
i have an intel i3-7100 with 8 gigs of ram. OS: debian with OMV (openmediavault)

**Comment by t-chab** [comment]
I'm sorry, but I can't reproduce the issue. 

Are you sure you're using a GUI client with the same version as amule daemon (2.3.2) ?

What happen if you use the new web ui (with environment variable _WEBUI_TEMPLATE_ set to _AmuleWebUI-Reloaded_)

Is the problem still there if you use another edonkey server ?
